pkg/validation/policy/validate.go

```diff
@@ -1675,31 +1630,6 @@ func validateNamespaces(validationFailureActionOverrides []kyvernov1.ValidationF
 	return nil
 }
 
-func checkForScaleSubresource(ruleTypeJson []byte, allKinds []string, warnings *[]string) {
```

Comment:
@realshuting - Do you happen to know why we check for `scale` and `status` subresources in the 1st place?

Reply:
This is for policy validation. For example, if one matches replicas doesn't include `scale` subresource.